chart/templates/mcp.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if and (ne .Values.config.OBOT_SERVER_MCPBASE_IMAGE "") (ne .Values.config.OBOT_SERVER_MCPNAMESPACE "") }}
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ .Values.config.OBOT_SERVER_MCPNAMESPACE }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  namespace: {{ .Values.config.OBOT_SERVER_MCPNAMESPACE }}
+  name: obot-mcp-role
+rules:
+  - apiGroups: [""]
+    resources: ["secrets", "services"]
+    verbs: ["create", "get", "list", "watch", "update", "patch", "delete"]
+  - apiGroups: ["apps"]
+    resources: ["deployments"]
+    verbs: ["create", "get", "list", "watch", "update", "patch", "delete"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  namespace: {{ .Values.config.OBOT_SERVER_MCPNAMESPACE }}
+  name: obot-mcp-rolebinding
+subjects:
+  - kind: ServiceAccount
+    name: obot
```

Comment:
instead of hardcoding the name, use `{{ include "obot.serviceAccountName" . }}`

Reply:
Crap, thanks. I meant to do that.